dream.py

```diff
@@ -46,23 +103,22 @@ def make_component_info(source_pos, sample_pos):
                    unit=sc.units.us)
 d['sample'] = sc.DataArray(coords={Dim.Tof: tofs})
 
-# Set some positions
-d.coords[Dim.Position].values[0] = [1, 2, 3]
-print(d.coords[Dim.Position].values[0])
+# check positions are reasonable
+print(d.coords[Dim.Position].values)
 
 # Add some events
 # Note: d.coords[Dim.Tof] gives the "dense" TOF coord, not the event-TOFs
 d['sample'].coords[Dim.Tof][Dim.Position, 0].values = np.arange(10)
 # The following should be equivalent but does not work yet, see scipp/#290
 # d['sample'].coords[Dim.Tof].values[1] = np.arange(10)
-print(d)
+#print(d)
 
 dspacing = sc.neutron.convert(d, Dim.Tof, Dim.DSpacing)
 print(dspacing)
 
 # Converting event data to histogram
-hist = sc.histogram(dspacing, dspacing.coords[Dim.DSpacing])
-print(hist)
+#hist = sc.histogram(dspacing, dspacing.coords[Dim.DSpacing])
+#print(hist)
```

Comment:
Is this not working any more with your changes, or why is it commented now?

Reply:
scipp complains about the dspacing bin edges not being sorted to make a histogram. I would look more into this once I have generated some data to be plotted.

Reply:
scipp complains about the dspacing bin edges not being sorted to make a histogram. I would look more into this once I have generated some data to be plotted.